platform/linux-dpdk/include/odp/api/plat/version_types.h

```diff
@@ -0,0 +1 @@
+../../../../../linux-generic/include/odp/api/plat/version_types.h
```

Comment:
@Bill-Fischofer-Linaro 
@lumag 

We seem agree that some technical debt (introduced long ago) needs to be addressed, but disagree on when and where it should be done.

Ideally, we would lay stubs for a new platform, refactor out the common code, rework parts of the build system, and start filling in the stubs from (already reviewed) code in odp-dpdk.git. This is usually a serial process, carried out by one person, since there are many dependencies as the code morphs into shape (filling stubs can run in parallel sometimes). From recent experiences, this will take at least 4-8+ weeks to go through upstream.

_But... there is something far more important than this..._

We need to practice empathy for the **5+** people in the community who are currently **blocked** from making progress on ODP Cloud development until such a new platform is added to odp.git repo. The most important issue right now should be establishing a _home_ for cloud-dev, not addressing technical debt. Can we agree?

This patch merges odp-dpdk.git repo into odp.git repo where /platform/linux-dpdk/ will foster ODP Cloud development. This is **exactly** what is needed, and what @GBalakrishna has been working towards.